tests/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
This needs to be generated with Python 3.10. `nox -e pip-compile-3.10(requirements)` will handle this automatically.

Reply:
@gotmax23 doh! I can't believe I did this...